generated/nifgen/nifgen_compilation_test.cpp

```diff
@@ -562,7 +562,7 @@ ViStatus SelfTest(ViSession vi, ViInt16* selfTestResult, ViChar selfTestMessage[
   return niFgen_self_test(vi, selfTestResult, selfTestMessage);
 }
 
-ViStatus SendSoftwareEdgeTrigger(ViSession vi, ViInt32 trigger, ViString triggerId)
+ViStatus SendSoftwareEdgeTrigger(ViSession vi, ViInt32 trigger, ViConstString triggerId)
```

Comment:
I'm not sure when this change got into the FGEN metadata but I believe it is correct. In the header file, it is a `ViConstString` for `triggerId` so this is more accurate and correct and avoids a casting to `ViString` in nifgen_service.cpp.